mwm-3-kotlin-1/presentation/PITCHME.md

```diff
@@ -1401,26 +1401,21 @@ class Dog() : Animal() {
 ### <span style="color: #00B8D4; text-transform: none; font-size:0.8em;">Singleton</span> <span style="text-transform: none; font-size:0.8em;">with object</span>
 
 ```kotlin
-object DateUtils {
-    fun getCurrentDate(): Date {
-        return Calendar.getInstance().time
+object ColorUtils {
+    fun getRedColor(): Int {
+        return Color.RED
```

Comment:
Nice better than before

---

Maybe show the usage / what we want to achieve in a slide before this one.

With only the title and:
`val red ColorUtils.getRedColor()`